geometry_msgs/Pose pose_goal
---
bool result